### DISCUSSION TITLE
Mac "locks up" with AlDente installed?

### DISCUSSION THREAD

**Discussion by g-a-c** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#627] [Q&A]
I have an Intel MacBook Pro 13" 2020, running up to date macOS (12.2.1 at the time of writing).

For several months now, I've had an intermittent issue with this machine, that sometimes it will "lock up" - by which I mean the display doesn't update for several seconds, but the mouse pointer still moves. If you wait somewhere between 10-30 seconds, then the screen goes black and kicks you back to a login screen, at which point you can log back in to a fresh user session with no apps open.

I originally thought that this was probably caused by the fact that my macOS installation was basically 7 years old from my old 2015 MBP, had only ever been upgraded (never reinstalled) and had been Time Machine restored several times (when I had to take the 2015 Mac in for repair, and when I bought the 2020). So a few weeks ago (Feb 23rd) I did a complete reinstall of macOS 12.2.1 from Internet Recovery, without Time Machine, and restored my data manually and reinstalled my apps fresh.

Today, I realised that I forgot to reinstall AlDente after this format a few weeks ago, so I reinstalled it. Less than 6 hours later, I got the "locking up" problem again.

I'm not saying that this is 100% caused by AlDente - but I find the timing very suspicious. It's always been an intermittent issue which makes it very hard to troubleshoot. So I'm curious if anyone else has ever had this happen?

**Comment by MatthiasKerbl** [comment]
Hi @g-a-c,

I have never heard about this issue before. Do you see any crash reports when it happens? Then we should be able to see if AlDente is the cause of this issue.

Cheers,
Matthias

**Reply by g-a-c** [reply]
There is no crash report dialog asking me to send information to Apple about a detected crash. There may be something buried in a log file - but I don't know enough about macOS to know which log file(s) would help here.

**Reply by MatthiasKerbl** [reply]
You can look for crash reports here:
Launch Finder, press Command + Shift + G, and type ~/Library/Logs/DiagnosticReports/ in the Go to Folder dialog box.

**Reply by g-a-c** [reply]
There's only one report in there from the approximate time I saw the issue, which is `SiriSearchFeedback-2022-03-13-211933.diag` and contains nothing immediately useful (I've anonymised the GUIDs just in case, but I doubt they're privacy-impacting in any way...):
```
{"timestamp":"2022-03-13 21:19:33.29 +0000","bug_type":"313","os_version":"macOS 12.2.1 (21D62)","incident_id":"A473E417-3FB6-4E64-98EA-0576AA3C5DD4"}
{"user_guid":"F5E9C970-5646-4A1C-BDD2-76B4950F7467","session_start":1647206373,"agent":"parsecd\/1.0 (MacBookPro16,2; macOS 12.2.1 21D62) Spotlight\/2183.14","country_code":"GB"}
```
There's several more of those so it looks like maybe that event gets generated every time the laptop sleeps/restarts, in which case the fact this was generated at 21:19 might be a symptom of my laptop locking up rather than a potential cause of it locking up.

**Reply by DamonHD** [reply]
In case it is relevant, it usually takes ~30s now for login/unlock screen to start flashing the password cursor after, eg, opening the lid.  That never used to happen before AlDente.

Rgds

Damon

**Reply by MatthiasKerbl** [reply]
> There's only one report in there from the approximate time I saw the issue, which is `SiriSearchFeedback-2022-03-13-211933.diag` and contains nothing immediately useful (I've anonymised the GUIDs just in case, but I doubt they're privacy-impacting in any way...):
> 
> ```
> {"timestamp":"2022-03-13 21:19:33.29 +0000","bug_type":"313","os_version":"macOS 12.2.1 (21D62)","incident_id":"A473E417-3FB6-4E64-98EA-0576AA3C5DD4"}
> {"user_guid":"F5E9C970-5646-4A1C-BDD2-76B4950F7467","session_start":1647206373,"agent":"parsecd\/1.0 (MacBookPro16,2; macOS 12.2.1 21D62) Spotlight\/2183.14","country_code":"GB"}
> ```
> 
> There's several more of those so it looks like maybe that event gets generated every time the laptop sleeps/restarts, in which case the fact this was generated at 21:19 might be a symptom of my laptop locking up rather than a potential cause of it locking up.

Unfortunately, it does not include too much information. Since no one has reported this issue before, I do not think it has anything to do with AlDente but of course, there is still the possibility. Please let me know when you have any new observations.

Cheers,
Matthias

**Reply by g-a-c** [reply]
> Since no one has reported this issue before, I do not think it has anything to do with AlDente but of course, there is still the possibility.

I would find it very strange as well, which is why I posted this as a discussion rather than an actual issue/bug, just to see if anyone else had a similar issue. I don't see how AlDente could cause it, but at the same time it's hard to deny the timing

**Reply by andrewcrook** [reply]
I have been getting SiriSearchFeedback diag files before I even installed AlDente 
I think its something to do with spotlight search.
